keyboards/sofle/rev1/rev1.c

```diff
@@ -1 +1,81 @@
 #include "sofle.h"
+
+#ifdef RGB_MATRIX_ENABLE
+#define RLO 36
+#define LLI(x) (x)
+#define LLP(x,y) {(x),(y)}
+#define RLI(x) (RLO+(x))
+#define RLP(x,y) {(224-(x)),((y))}
+  // Physical Layout
+  // Columns
+  // 0  1  2  3  4  5  6  7  8  9  10 11 12 13
+  //                                           ROWS
+  // 12 13 22 23 32 33       33 32 23 22 13 12  0
+  //    02    03    04       04    03    02
+  // 11 14 21 24 31 34       34 31 24 21 14 11  1
+  //                01       01
+  // 10 15 20 25 30 35       35 30 25 20 15 10  2
+  //
+  // 09 16 19 26 29 36       36 29 26 19 16 09  3
+  //
+  //     08 17 18 27 28     28 27 18 17 08      4
+  //    07    06    05       05    06    07
+
+led_config_t g_led_config = {
+    {
+        {  LLI(11),  LLI(12),  LLI(21),  LLI(22),  LLI(31),  LLI(32) },
```

Comment:
Please change this to the normal format for g_led_config.

Reply:
Is that by removing the macros?

Reply:
Yeah.

Reply:
Is it okay like this? Should I also remove the comments?